SpatialGDK/Documentation/content/project-setup.md

```diff
@@ -0,0 +1,53 @@
+# Set up the Third-Person Shooter game or Test Suite 
+
+Use this guide to set up the [Third-Person Shooter game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) or the [Test Suite](https://github.com/spatialos/UnrealGDKTestSuite) GDK projects. 
```

Comment:
Might be worth putting a note readable by users that the doc is not in the ToC hierarchy and only linked to from the Third-Person Shooter game and Test Suite repo readmes so if you navigate away from this page, you won't be able to find it in the ToC - maybe best to book mark it?

Otherwise peeps might get lost and confused if they go to Getting Started and Get and build dependencies.

Alternative is to put it in the ToC somewhere - but as we are de-emphasising these we probs don't want to?